src/FtpClient.php

```diff
@@ -598,11 +692,7 @@ private function throwIfNotConnected() : void
      */
     private function call(bool $throw, callable $function, ...$parameters)
     {
-        if ($throw) {
-            set_error_handler([$this, 'errorHandlerException']);
-        } else {
-            set_error_handler([$this, 'errorHandlerIgnore']);
-        }
+        set_error_handler([$this, $throw ? 'errorHandlerException' : 'errorHandlerIgnore']);
```

Comment:
I actually don't like this, as it will make static analysis harder (I doubt that PHPStorm, Psalm etc. will be able to infer the method from a dynamic callable with a ternary operator. I prefer to keep it this way.

Reply:
Okay, I'll fix it.